devops/parachain/docker-compose.yml

```diff
@@ -72,6 +72,7 @@ services:
     image: ${PARACHAIN_DOCKER_IMAGE:-mangatasolutions/mangata-node:latest}
     environment:
       - ${RUST_LOG:-info}
+      - LD_PRELOAD=${LD_PRELOAD}
```

Comment:
maybe `${LD_PRELOAD:-""}` to stick to single convention ?

Reply:
lets see if this works :)

Reply:
It does not, when running a command like this,

`export MANGATA_NODE_VERSION=442771a8aff53309db52f472fea8ae593c71b76a.tar; docker-compose -f ./docker-compose.yml down ; LD_PRELOAD=/mangata/libbytehound.so  docker-compose -f ./docker-compose.yml up`
It does not activate the profiler. Leaving it as it was before.